MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/text/LocalGlyphRasterizer.java

```diff
@@ -31,7 +31,7 @@ of the bitmap (y: 20) with some buffer around the edge
 
     paint = new Paint();
     paint.setAntiAlias(true);
-    paint.setTextSize(24);
+    paint.setTextSize(22);
```

Comment:
Will this cause an issue if a developer sets a larger font size via the app or via studio?

Reply:
What's the text size do you set in your style? Can you set it to a very large one to help me verify it?

Reply:
@tsuz @Chaoba `24pt` is a base font size across whole mapbox ecosystem, server and client side. This value should not be changed.

We use SDF rasterezed glyphs, they can be up/downscaled without much artifacts. I don't see how setting very large text size in studio would help here.

I think the issue is that for some reason, Android renders CJK glyph too close to a bitmap border (we expect to have have 3px padding around rendered glyph)
35x35px buffer comes from 24px + 3px border * 2 + 1px glyph padding * 2 + safe margin.

@Chaoba could you increase buffer size and see if truncation is fixed? You can also save bitmap locally and see if there is a truncation in SDF glyph.